server/views/pages/personal-details.njk

```diff
@@ -238,6 +238,10 @@
             <p>
             <a href="/case/{{ crn }}/handoff/oasys">View sentence plan in OASys</a>
             </p>
+
+            <p>
+            <a href="{{ sentencePlan }}">View sentence plan in UI</a>
```

Comment:
`sentencePlan` doesnt resolve to anything here. We need to add a ts function like this in `server/utils/utils.ts`

```
export const sentencePlanLink = () => {
  return `${config.sentencePlan.link}`
}
```
then add the function to the global nunJucks in `server/utils/nunJucksSetup.ts`

`njkEnv.addGlobal('sentencePlanLink', sentencePlanLink())`

Then the function can be called from the nunjucks template

`<a href="{{ sentencePlanLink }}" target="_blank">View sentence plan in UI</a>`

Also i think this should open in a new window hence i have specified `target="_blank"` above in the href